torch_xla/core/xla_model.py

```diff
@@ -548,12 +546,12 @@ def all_gather(value, dim=0, groups=None, output=None, pin_layout=True):
     new_token = torch_xla._XLAC._xla_all_gather_out(output, value, token, dim,
                                                     shard_count, groups or [],
                                                     pin_layout)
-    devctx.all_reduce_token = new_token
+    torch_xla._XLAC._set_all_reduce_token(new_token)
     return output
 
   result = torch_xla._XLAC._xla_all_gather(value, token, dim, shard_count,
                                            groups or [], pin_layout)
-  devctx.all_reduce_token = result[1]
+  torch_xla._XLAC._set_all_reduce_token(result[1])
```

Comment:
hmm, I guess in the long term if we don't even set the token in python, there is no need to return the token to the python layer? We can look into this later, maybe it is still cleaner this way.

Reply:
Yea, I'm just too lazy to make the changes to all cc ops to remove the token in the python layer.